python/lsst/galaxy_shear/processShearTest.py

```diff
@@ -160,6 +212,17 @@ def run(self, dataRef):
             calib = lsst.afw.image.Calib()
             calib.setFluxMag0((3531360874589.57, 21671681149.139))
             exp.setCalib(calib)
+
+            #  Add a real footprint
+            if self.config.footprintSize is None:
+                try:
+                    footprints = lsst.meas.algorithms.SourceDetectionTask().detectFootprints(exp,
+                                 sigma=5.0).positive.getFootprints()
+                    source.setFootprint(footprints[0])
```

Comment:
You may also want to just set the error flag if the number of Footprints is != 1.